Resolver/FormAnnotationResolver.php

```diff
@@ -159,12 +159,12 @@ protected function getBuiltObject(
         $entity = $request->attributes->get($annotation->getEntity());
 
         /**
-         * Creates form object from type
+         * Creates form object from type.
          */
-        $form = $formFactory->create($type, $entity);
+        $form = $formFactory->create(get_class($type), $entity);
```

Comment:
Yes XD exactly XD
This is one of the changes I've need to do for 3.0.0. Class has to be passed instead of the object itself (I don't really like this change... but, anyway...)